<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderHistoryDetailsWindow</class>
 <widget class="QDialog" name="OrderHistoryDetailsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XKOMX</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="saDevices">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>710</width>
        <height>330</height>
       </rect>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>160</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vlDevices"/>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lbDate">
       <property name="text">
        <string>XX.XX.XXXX</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbTotalCost">
       <property name="text">
        <string>XXXXX PLN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
